pkgs/servers/zoneminder/default.nix

```diff
@@ -78,19 +78,21 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "zoneminder";
-  version = "1.32.3";
+  version = "1.34.3";
 
   src = fetchFromGitHub {
     owner  = "ZoneMinder";
     repo   = "zoneminder";
     rev    = version;
-    sha256 = "1sx2fn99861zh0gp8g53ynr1q6yfmymxamn82y54jqj6nv475njz";
+    sha256 = "0jp7950v36gxxzkwdp5i0312s26czhfsl5ixdxfzn21cx31hhlg0";
   };
 
   patches = [
     ./default-to-http-1dot1.patch
-    # Explicitly link with dynamic linking library to fix build
-    ./link-with-libdl.patch
+    (substituteAll {
+      src = ./0001-Don-t-use-file-timestamp-in-cache-filename.patch;
+      srcHash = src.outputHash;
```

Comment:
I guess, it makes more sense to tag cache with ``` `basename $out` ``` instead of just `$src`.

Reply:
OK. I've tested and pushed a new commit for this change.